.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - '6'
+
+dist: trusty
+sudo: required
```

Comment:
They provide Ubuntu 12 and 14, I want to make sure it is 14.
Needed sudo to be able to create the db.

Also there is another reason: https://docs.travis-ci.com/user/ci-environment/
Here they say:
```
Sudo-enabled (a sudo enabled, full VM per build). Ubuntu 12 or 14.

Container-based (Fast boot time environment in which sudo commands are not available). Ubuntu 12
```
As our production usage is non-docker I tried to make a test env that is not container based.